subsys/net/ip/net_core.c

```diff
@@ -65,12 +66,66 @@ void net_analyze_stack(const char *name, const char *stack, size_t size)
 }
 #endif /* CONFIG_INIT_STACKS */
 
+#if defined(CONFIG_NET_SOCKETS_RAW)
+static inline void process_raw_data(struct net_pkt *pkt)
+{
+	struct net_context *cont;
+	struct net_pkt *raw_pkt;
+	int ret;
+
+	cont = net_context_find(AF_PACKET, SOCK_RAW, ETH_P_ALL);
+	if (!cont) {
+		return;
+	}
+
+	raw_pkt = net_pkt_clone(pkt, K_MSEC(100));
```

Comment:
Use a macro for the timeout, as everywhere in the core stack. Something like BUF_TIMEOUT